test/cell-test.js

```diff
@@ -583,17 +611,17 @@ describe('Cell',function(){
     describe('top line',function(){
       it('will draw the top left corner when x=0,y=0',function(){
         cell.x = cell.y = 0;
-        expect(cell.draw('top')).to.equal('┌───────');
+        expect(cell.draw('top')).to.equal('╔═══════');
```

Comment:
You shouldn't change existing tests unless you are introducing a breaking change. That defeats the purpose as we lose confidence that this change isn't creating a regression for existing users.

Reply:
How do the changes made to `test/cell-test.js` allow introducing a regression?

As far as I can tell, the tests in that file behave exactly as before with the addition of testing the additional features of this PR. Namely, configuring different outer and inner characters.

Reply:
@jamestalmage Should I revert the changes to the existing tests?

As stated before, I believe the changes to the tests don’t change them in their intended behavior. They only add further coverage of the features that were added.